harmonisation/intro.md

```diff
@@ -0,0 +1,21 @@
+# Introduction to harmonisation
+
+For DiSSCo to provide be fully machine actionable units (FDOs) there needs to be a standardized datamodel.
```

Comment:
'to provide be fully machine actionable..' this reads weird

Reply:
Comma added